docs/organizations/using-a-coding-standard.md

```diff
@@ -0,0 +1,82 @@
+
+Use the organization coding standard to ensure that Codacy analyzes all your repositories with the same tool and code pattern settings. For example, configure the organization coding standard to ensure that all your repositories and teams are following the same global security rules or coding conventions.
+
+If you then customize the tool and code patterns for a specific repository, Codacy warns you that the repository will stop following the coding standard and asks for your confirmation.
+
+## Creating a coding standard {: id="creating"}
+
+Organizations on Codacy can have one coding standard. To create a coding standard for your organization:
+
+1.  Open your organization **Coding standards** page, tab **Coding standards**.
+
+1.  Name your coding standard and click **Create standard**.
+
+    Optionally, select a repository that Codacy will use as a baseline to bootstrap the tool and pattern configurations for the new coding standard. This is useful if you already have a repository with code patterns configured and that you wish to use as a template for the coding standard.
+
+    ![Creating a new coding standard](images/coding-standard-create.png)
+
+1.  Select all the programming languages that the new coding standard will cover and click **Next: Tools and patterns**.
+
+    The coding standard will only include configurations for the tools that support at least one of the selected languages.
+
+    ![Selecting the languages for the coding standard](images/coding-standard-select-languages.png)
+
+1.  Configure the tools and patterns of the coding standard and click **Next: Select and apply to repositories**.
+
+    For each tool:
+
+    -   Toggle if Codacy will run the tool when analyzing your code
+    -   Configure the code patterns that the tool will use
+
+    You can use the filters to help you find the relevant tools and code patterns.
+
+    ![Configuring the tools and patterns for the coding standard](images/coding-standard-configure-tools.png)
+
+1.  Select existing repositories that should follow the new coding standard and click **Save and apply standard**.
+
+    Codacy will use the new coding standard when analyzing the next commit on each selected repository.
+
+    ![Applying the coding standard to repositories](images/coding-standard-apply.png)
+
+## Setting a coding standard as default {: id="set-default"}
+
+New repositories in your organization will automatically follow the default coding standard.
+
+To set a coding standard as default:
+
+1.  Open your organization **Coding standards** page, tab **Coding standards**.
+
+1.  Toggle **Make default** on the coding standard card.
+
+    ![Setting a coding standard as the default](images/coding-standard-set-default.png)
```

Comment:
```suggestion
```

We will release without this https://docs.google.com/document/d/1UBXdsITNEf380UhjwckKaxT6z1ubANTQQwip0QG17Nk/edit?disco=AAAARRx3SIM

Reply:
Thanks for the heads up @inesgomas. :bow: In the meanwhile, I commented out these sections so it's easy to add them back and review them later. :+1: